content/articles/changing-email.md

```diff
@@ -15,16 +15,6 @@ categories:
 
 ---
 
-## A note about accounts
-
-When you initially sign up as a user in our system, we create an account bound to that user. We call this the Primary Account, and the email address is the same as the username. If you change the email address for that user, the account email address will also be updated. That way customers with a single account only have to update their email in one place. Follow the instructions in [Changing User Email](#changing-the-user-email) if this is the case for you.
-
-Extra accounts have their own email addresses that can be changed ad-hoc.
-
-<warning>
-System limitations do not currently allow Primary Accounts to be deleted.
-</warning>
-
 ## Changing the account email
```

Comment:
I'm confused here and think I am missing something. 

I thought we were changing this so that you only have a user email. Why would there be an account email as well?

Reply:
Never mind. I just looked at https://github.com/dnsimple/dnsimple-app/pull/27073/files and see what you're doing. The two are no longer connected so updating one does not update the other.

Reply:
That's exactly it. Thanks for taking a look @stephanoodle ❤️

Reply:
Perhaps @itsalyse can a find a place to clarify this: 🥺 
1. Account email is the one that receives notifications (eg. expiring domains, billing, etc).
2. User email is just for logging in (and user-specific notifications like SSO, reset password).